lib/fMailbox.php

```diff
@@ -348,15 +348,15 @@ private static function parseHeaders($headers, $filter = null)
 		$multi_email_fields     = array('to', 'cc');
 		$additional_info_fields = array('content-type', 'content-disposition');
 
-		$headers = array();
+		$parsedHeaders = array();
```

Comment:
$parsed_headers matches the code style around.

Reply:
Oops!

Reply:
Fixed!